tests/python/gaia-ui-tests/gaiatest/apps/homescreen/app.py

```diff
@@ -72,47 +67,9 @@ def open_context_menu(self):
         return ContextMenu(self.marionette)
 
     def move_app_to_position(self, app_position, to_position):
-        app_elements = self.app_elements
-
-        self.marionette.execute_script(
-            'arguments[0].scrollIntoView(false);', [app_elements[app_position]])
-
-        Actions(self.marionette).\
-            press(app_elements[app_position]).\
-            wait(3).\
-            move(app_elements[to_position]).\
-            wait(1).\
-            release().\
-            wait(1).\
-            perform()
-
-    def move_to_divider(self, app_position, divider_position):
-        app_element = self.app_elements[app_position]
-        separator_element = self.marionette.find_elements(*self._divider_separator_locator)[divider_position]
-
-        self.marionette.execute_script(
-            'arguments[0].scrollIntoView(false);', [app_element])
-
-        Actions(self.marionette).\
-            press(app_element).\
-            wait(3).\
-            move(separator_element).\
-            wait(1).\
-            release().\
-            wait(1).\
-            perform()
-
-    @property
-    def is_edit_mode_active(self):
-        return self.is_element_present(*self._edit_mode_locator)
-
-    def tap_edit_done(self):
-         element = self.marionette.find_element(*self._exit_edit_mode_locator)
-         Wait(self.marionette).until(lambda m: element.is_displayed())
-         element.tap()
-         Wait(self.marionette).until(lambda m: not element.is_displayed())
-         Wait(self.marionette).until(expected.element_not_present(
-             *self._edit_mode_locator))
+        app = self.app_elements[app_position]
+        destination_element = self.app_elements[to_position].root_element
+        app.move_to(destination_element)
 
     def tap_collection(self, collection_name):
```

Comment:
This method can also be removed (I forgot to remove this in another pull request)